DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -103,88 +107,145 @@ public IActionResult AddRegistrationPromptSelectPrompt(AddRegistrationPromptSele
                 return View(model);
             }
 
+            UpdateTempDataWithSelectPromptModelValues(model);
+
+            return RedirectToAction("AddRegistrationPromptConfigureAnswers");
+        }
+
+        [HttpGet]
+        [Route("Add/ConfigureAnswers")]
+        [ServiceFilter(typeof(RedirectEmptySessionData<AddRegistrationPromptData>))]
+        public IActionResult AddRegistrationPromptConfigureAnswers()
+        {
             var data = TempData.Peek<AddRegistrationPromptData>()!;
-            data.SelectPromptViewModel = model;
-            TempData.Set(data);
+            var viewModel = data.ConfigureAnswersViewModel;
 
-            // TODO: HEEDLS-453 - redirect to next page
-            return RedirectToAction("Index");
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        [Route("Add/ConfigureAnswers")]
+        [ServiceFilter(typeof(RedirectEmptySessionData<AddRegistrationPromptData>))]
+        public IActionResult AddRegistrationPromptConfigureAnswers(
+            RegistrationPromptAnswersViewModel model,
+            string action
+        )
+        {
+            if (action.StartsWith(DeleteAction) && TryGetAnswerIndexFromDeleteAction(action, out var index))
+            {
+                return RegistrationPromptAnswersPostRemovePrompt(model, index, true);
+            }
+
+            return action switch
+            {
+                NextAction => AddRegistrationPromptConfigureAnswersPostNext(model),
+                AddPromptAction => RegistrationPromptAnswersPostAddPrompt(model, true),
+                _ => RedirectToAction("Error", "LearningSolutions")
+            };
         }
 
-        private bool TryGetAnswerIndexFromAction(string action, out int index)
+        private IActionResult EditRegistrationPromptPostSave(EditRegistrationPromptViewModel model)
         {
-            return int.TryParse(action.Remove(0, 6), out index);
+            IgnoreAddNewAnswerValidation();
+
+            if (!ModelState.IsValid)
+            {
+                return View(model);
+            }
+
+            customPromptsService.UpdateCustomPromptForCentre
+                (User.GetCentreId(), model.PromptNumber, model.Mandatory, model.OptionsString);

```

Comment:
Ah, could we use the same formatting in this case as we do for the function param declaration?